### PR TITLE
Changes for credencials texts

### PR DESCRIPTION
**Pull request by adfonseca**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [x] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [x] Requires translations.

### CODE REVIEW

File: guides/sdks/official/php.es.md [hunk 1]
```diff
@@ -10,39 +10,45 @@ Nuesto SDK es compatible con las versiones de PHP 5.6 o superior.
...
   require __DIR__  . '/vendor/autoload.php';
   ```
 
-2. Configura tus credenciales
+2) Configura tus credenciales
   
-  Existen 2 tipos de credenciales:
+  Existen dos tipos de credenciales:
 
   * **Para el Checkout Mercado Pago:**
```
Comment (mminestrelli) [inline guides/sdks/official/php.es.md:33]: No deberíamos cambiar esto? ya no es más así. Está dentro del scope @pamalv7?

File: guides/sdks/official/ruby.pt.md [hunk 2]
```diff
@@ -101,18 +101,24 @@ puts result
 ```
 
 <a name="custom-checkout"></a>
-## Checkout customizado
+## Checkout API
```
Comment (mminestrelli) [inline guides/sdks/official/ruby.pt.md:104]: @pamalv7 no es consistente con el naming del producto

File: guides/sdks/official/php.es.md (hunk 1 above)
Comment (adfonseca) [reply guides/sdks/official/php.es.md:33]: Está correcto Mauri. Voy cambiar.

File: guides/sdks/official/ruby.pt.md (hunk 2 above)
Comment (adfonseca) [reply guides/sdks/official/ruby.pt.md:104]: Este esta diferente de los otros. Pedi información del equipo de SDKs para continuarmos.

File: guides/marketplace/api/introduction.es.md [hunk 3]
```diff
@@ -24,13 +24,17 @@ Después de crear la aplicación, sólo es necesario ejecutar el segundo y terce
...
+La clave pública, o **Public key**, te sirve para acceder a los recursos que necesita tu frontend. Vas a poder recolectar los datos de las tarjetas de crédito y convertirlos en un token representativo que puedes guardar de forma segura en tus servidores para crear un pago.
 
-La **credencial privada**, o *access\_token*, se utiliza para todas las otras llamadas a las APIs, como procesar un pago, realizar reembolsos, almacenar tarjetas y más. Las claves privadas deben ser mantenidas **confidencialmente** en tus servidores de _backend_ y nunca deben ser publicadas.
+Y la clave privada, o **Access token**, te va a permitir llamar al resto de las APIs. Por ejemplo, cómo procesar un pago, realizar un reembolso o almacenar tarjetas.
 
-> Haciendo click en el botón "renovar credenciales" obtienes pares nuevos y los anteriores dejan de funcionar. Usa esto sólo cuando creas que tus credenciales privadas han sido vulneradas o por cuestiones de seguridad, similares al cambio de contraseña, cada cierto período de tiempo. Recuerda reemplazar las credenciales en tu aplicación para que siga funcionando.
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/marketplace/api/introduction.es.md:36]:
```suggestion
```

File: guides/marketplace/web-checkout/introduction.en.md [hunk 4]
```diff
@@ -24,8 +24,16 @@ After creating the application, you only need to take the second and third steps
...
 
-Similar to the Checkout of Payments, you have a pair of private keys to connect to the API. You can find these keys in the [credentials section of your account]([FAKER][CREDENTIALS][URL]).
+Your <a href="[FAKER][CREDENTIALS][URL]" target="_blank"> credentials</a> are the **keys we provide so you can configure your integrations.**.
 
 The **private keys** (also known as `access_token`) are used for all the requests to the APIs, such as processing payments, refunds, etc. The private keys must be kept **confidentially** in your backend servers and should never be published.
 
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/marketplace/web-checkout/introduction.en.md:34]:
```suggestion
```

File: guides/marketplace/api/introduction.pt.md [hunk 5]
```diff
@@ -25,13 +25,18 @@ Após criar a aplicação, você só precisa executar o segundo e o terceiro pas
...
+representativo que pode ser guardado de forma segura em seus servidores para criar um pagamento. 
 
-A **chave privada**, ou *access_token*, é utilizada para todas as requisições realizadas às APIs, tais como processamento de pagamentos, reembolsos, armazenamento de cartões, etc. As chaves privadas devem ser mantidas **confidencialmente** em seus servidores de backend e nunca devem ser publicadas.
+A chave privada, ou **Access token**, é utilizada para todas as requisições realizadas às APIs, tais como processamento de pagamentos, reembolsos, armazenamento de cartões, etc. As chaves privadas devem ser mantidas confidencialmente em seus servidores de backend e nunca devem ser publicadas.
 
-> Ao clicar no botão “renovar credenciais”, você obterá novos pares de chaves e as anteriores deixarão de funcionar. Faça isso apenas se desconfiar que suas chaves privadas foram violadas ou por questões de segurança, semelhante à alteração de senha, a cada período de tempo. Lembre-se de substituir as credenciais em sua aplicação para mantê-la funcionando.
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/marketplace/api/introduction.pt.md:38]:
```suggestion
```

File: guides/marketplace/web-checkout/introduction.es.md [hunk 6]
```diff
@@ -26,13 +26,19 @@ Después de crear la aplicación, sólo es necesario ejecutar el segundo y terce
...
+La clave pública, o **Public key**, te sirve para acceder a los recursos que necesita tu frontend. Vas a poder recolectar los datos de las tarjetas de crédito y convertirlos en un token representativo que puedes guardar de forma segura en tus servidores para crear un pago.
 
-Antes de utilizar el Access Token se deberá completar el formulario de "Quiero ir a producción".
+Y la clave privada, o **Access token**, te va a permitir llamar al resto de las APIs. Por ejemplo, cómo procesar un pago, realizar un reembolso o almacenar tarjetas.
 
-Al estar utilizando el Checkout Mercado Pago, las credenciales que se deberán utilizar para la asociación de los vendedores a tu Marketplace (2º Paso) son el Client_id y Client_secret de la sección de Checkout Básico en la página de credenciales. 
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/marketplace/web-checkout/introduction.es.md:38]:
```suggestion
```

File: guides/payments/mobile-checkout/latest/introduction.en.md [hunk 7]
```diff
@@ -39,10 +39,15 @@ Offer the best payment experience to your users, on Android or iOS.
...
 1. Include the SDK in your project.
-2. Enter your credentials and create the payment preference.
+2. Enter your <a href="[FAKER][CREDENTIALS][URL]" target="_blank"> credentials</a> and create the payment preference.
 3. Start the payment process with a button on your application.
 4. Get to know when a payment is made with the notifications we send you.
 
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/payments/mobile-checkout/latest/introduction.en.md:49]:
```suggestion
```

File: guides/payments/mobile-checkout/latest/introduction.es.md [hunk 8]
```diff
@@ -39,10 +39,15 @@ Ofrece a tus usuarios la mejor experiencia de pago, tanto en Android como en iOS
...
 1. Incluye el SDK en tu proyecto.
-2. Coloca tus credenciales y crea la preferencia de pagos.
+2. Coloca tus <a href="[FAKER][CREDENTIALS][URL]" target="_blank"> credenciales </a> y crea la preferencia de pagos.
 3. Inicia el proceso de pago desde un botón en tu aplicación.
 4. Enterate del pago escuchando las notificaciones que te enviamos.
 
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/payments/mobile-checkout/latest/introduction.es.md:49]:
```suggestion
```

File: guides/payments/mobile-checkout/latest/introduction.pt.md [hunk 9]
```diff
@@ -40,10 +40,16 @@ Ofereça aos seus usuários a melhor experiência de pagamento, tanto no Android
...
 1. Inclua o SDK em seu projeto.
-2. Insira suas credenciais e crie a preferência de pagamentos.
+2. Insira suas <a href="[FAKER][CREDENTIALS][URL]" target="_blank"> credenciais </a> e crie a preferência de pagamentos.
 3. Inicie o processo de pagamento a partir de um botão em sua aplicação.
 4. Receba as notificações de pagamento que lhe enviamos.
 
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/payments/mobile-checkout/latest/introduction.pt.md:50]:
```suggestion
```

File: guides/plugins/unofficial/bis.en.md [hunk 10]
```diff
@@ -30,7 +30,14 @@ Com a conta do MercadoPago já criada, você deve informar os dados abaixo para
 
 #### Credenciais
 
-[Public Key e Access Token]([FAKER][CREDENTIALS][URL])
+[Public key e Access token]([FAKER][CREDENTIALS][URL])
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/plugins/unofficial/bis.en.md:38]:
```suggestion
```

File: guides/plugins/unofficial/bis.es.md [hunk 11]
```diff
@@ -28,7 +28,13 @@ Con la cuenta de MercadoPago ya creada, debe ingresar los datos a continuación
 
 #### Credenciales
 
-[Public Key y Access Token](([FAKER][CREDENTIALS][URL]))
+[Public key y Access token](([FAKER][CREDENTIALS][URL]))
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/plugins/unofficial/bis.es.md:36]:
```suggestion
```

File: guides/plugins/unofficial/bis.pt.md [hunk 12]
```diff
@@ -30,7 +30,13 @@ Com a conta do MercadoPago já criada, você deve informar os dados abaixo para
 
 #### Credenciais
 
-[Public Key e Access Token]([FAKER][CREDENTIALS][URL])
+[Public key e Access token]([FAKER][CREDENTIALS][URL])
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/plugins/unofficial/bis.pt.md:38]:
```suggestion
```

File: guides/plugins/unofficial/spreedly.en.md [hunk 13]
```diff
@@ -11,7 +11,13 @@ The available Spreedly methods with Mercado Pago are:
 - Void
 - Verify
 
-To integrate Spreedly you need to obtain and setup your Mercado Pago keys.You can find these keys in the [credentials section of your account]([FAKER][CREDENTIALS][URL]).
+To integrate Spreedly you need to obtain and setup your Mercado Pago [credentials]([FAKER][CREDENTIALS][URL]) Public key and Access token.
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/plugins/unofficial/spreedly.en.md:19]:
```suggestion
```

File: guides/plugins/unofficial/spreedly.es.md [hunk 14]
```diff
@@ -10,7 +10,13 @@ Los metodos disponibles de Spreedly con Mercado Pago son:
 - Void
 - Verify
 
-Para la integración con Spreedly es necesario obtener y configurar las credenciales de Mercado Pago que podes obtener desde el siguiente [link]([FAKER][CREDENTIALS][URL])
+Para la integración con Spreedly es necesario obtener y configurar las [credenciales]([FAKER][CREDENTIALS][URL]) Public key y Access token de Mercado Pago.
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/plugins/unofficial/spreedly.es.md:18]:
```suggestion
```

File: guides/plugins/unofficial/spreedly.pt.md [hunk 15]
```diff
@@ -10,7 +10,13 @@ Os métodos disponíveis de Spreedly com Mercado Pago são:
 - Void
 - Verify
 
-Para integração com Spreedly é necessário obter e configurar as credenciais de Mercado Pago que pode obter à partir do seguinte link [credentials section of your account]([FAKER][CREDENTIALS][URL]).
+Para integração com Spreedly é necessário obter e configurar as [credenciais]([FAKER][CREDENTIALS][URL]) Public key e Access token do Mercado Pago.
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/plugins/unofficial/spreedly.pt.md:18]:
```suggestion
```

File: guides/plugins/unofficial/virtue-mart.en.md [hunk 16]
```diff
@@ -171,7 +171,13 @@ The module of Mercado Pago to VirtueMart is integrated with the features and pay
 
   ![Settings credentials - Transparent checkout - Ticket](/images/virtuemart-credentials_custom_ticket.png) <br />
 
-  Find your **access_token** [Credentials section]([FAKER][CREDENTIALS][URL]).  
+  Find your **Access token** in the [Credentials section]([FAKER][CREDENTIALS][URL]).  
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/plugins/unofficial/virtue-mart.en.md:179]:
```suggestion
```

File: guides/plugins/unofficial/virtue-mart.es.md [hunk 17]
```diff
@@ -172,7 +172,13 @@ Puedes obtener tu **Public Key** y **Access Token** [en la sección credenciales
 
 ![Settings credentials - Transparent checkout - Ticket](/images/virtuemart-credentials_custom_ticket.png) <br />
 
-Puedes obtener tu **Public Key** y **Access Token** [en la sección credenciales]([FAKER][CREDENTIALS][URL]).
+Puedes obtener tu **Access Token** [en la sección credenciales]([FAKER][CREDENTIALS][URL]).
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/plugins/unofficial/virtue-mart.es.md:180]:
```suggestion
```

File: guides/plugins/unofficial/virtue-mart.pt.md [hunk 18]
```diff
@@ -145,12 +145,18 @@ Encontre seu **Public Key** e **Access Token** na [seção Credenciais]([FAKER][
...
+6. Agora configure suas credenciais. Para fazer isso, preencha sua **Public key** e **Access token** na sessão de Credenciais.
 
 ![Settings credentials - Transparent checkout - Ticket](/images/virtuemart-credentials_custom_ticket.png) <br />
 
 Encontre seu **Access Token** na [seção Credenciais]([FAKER][CREDENTIALS][URL]).
 
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/plugins/unofficial/virtue-mart.pt.md:157]:
```suggestion
```

File: guides/plugins/woocommerce/integration.es.md [hunk 19]
```diff
@@ -12,9 +12,15 @@ Conecta una cuenta de Mercado Pago al módulo para capturar los cobros de tus ve
...
 
 1. Crea una [cuenta vendedor](https://www.mercadopago.com.ar/registration-company?confirmation_url=https%3A%2F%2Fwww.mercadopago.com.ar%2Fcomo-cobrar) en Mercado Pago si todavía no tienes una.
-2. Obtén el par de [credenciales](https://www.mercadopago.com.ar/developers/es/guides/localization/credentials) Access Token y Public Key y pégalas en los campos de Producción y Pruebas que encontrarás en la configuración del módulo.
+2. Obtén el par de <a href="[FAKER][CREDENTIALS][URL]" target="_blank"> credenciales </a> Access Token y Public Key y pégalas en los campos de Producción y Pruebas que encontrarás en la configuración del módulo.
 3. Homologa la cuenta para [ir a Producción](https://www.mercadopago.com.ar/developers/es/guides/payments/api/goto-production/) y recibir el dinero de tus ventas en Mercado Pago.
 
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/plugins/woocommerce/integration.es.md:21]:
```suggestion
```

File: guides/plugins/woocommerce/integration.pt.md [hunk 20]
```diff
@@ -12,9 +12,15 @@ Conecte uma conta do Mercado Pago ao módulo para capturar os recebimentos das s
...
 
 1. Crie uma [conta de vendedor](https://www.mercadopago.com.br/registration-company?confirmation_url=https%3A%2F%2Fwww.mercadopago.com.br%2Fcomo-cobrar) no Mercado Pago caso ainda não tenha uma.
-2. Crie algumas [credenciais](https://www.mercadopago.com.br/developers/pt/guides/localization/credentials)) Access Token e Public Key e cole-as nos campos de Produção e Testes que estarão na configuração do módulo.
+2. Crie algumas <a href="[FAKER][CREDENTIALS][URL]" target="_blank"> credenciais </a> Access Token e Public Key e cole-as nos campos de Produção e Testes que estarão na configuração do módulo.
 3. Homologue a conta para [ir à Produção](https://www.mercadopago.com.br/developers/pt/guides/payments/api/goto-production/) e receber o dinheiro das suas vendas no Mercado Pago.
 
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/plugins/woocommerce/integration.pt.md:21]:
```suggestion
```

File: guides/qr-code/general-considerations/pre-requisites.es.md [hunk 21]
```diff
@@ -17,14 +17,20 @@ Sabemos que algunos términos son nuevos. Antes de empezar, te los dejamos a man
...
 | `COLLECTOR_ID` | ID de la cuenta de Mercado Pago. Se compone por los dígitos posteriores al guión medio del `access_token`. También lo puedes encontrar como _`USER_ID`_. |
 | `SPONSOR_ID` |  Es el `collector_ID` del usuario proveedor del sistema integrado con Mercado Pago. El `sponsor_ID` no puede ser igual al `collector_id`. |
 | Sucursal | Es una **tienda física** en la que tus clientes pueden adquirir tus productos o servicios. Puedes tener varias sucursales en una misma cuenta. |
 | Caja | Es un **punto de venta** que existe en una sucursal o tienda física. Cada caja tendrá vinculado un código QR unívoco. |
 | Orden | Es el pedido realizado por tu cliente. Contiene un listado de productos con su monto asociado.
 
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/qr-code/general-considerations/pre-requisites.es.md:31]:
```suggestion
```

File: guides/qr-code/general-considerations/pre-requisites.pt.md [hunk 22]
```diff
@@ -25,6 +25,12 @@ Sabemos que alguns termos são novos. Antes de começar, os deixamos perto de su
 | Caixa | É um **ponto de venda** que existe numa sucursal ou loja física. Cada caixa terá um código QR unívoco vinculado.  |
 | Ordem | É o pedido realizado pelo seu cliente. Contém uma relação de produtos com seu valor associado. 
 
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/qr-code/general-considerations/pre-requisites.pt.md:31]:
```suggestion
```

File: guides/sdks/official/dotnet.es.md [hunk 23]
```diff
@@ -25,26 +25,33 @@ Nuesto SDK es compatible con las versiones de .NET 4.5 o superior.
...
 
 ```csharp
 MercadoPago.SDK.AccessToken = "ENV_ACCESS_TOKEN";
 ```
-3. Utiliza los recursos
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/sdks/official/dotnet.es.md:51]:
```suggestion
```

File: guides/sdks/official/dotnet.pt.md [hunk 24]
```diff
@@ -25,27 +25,33 @@ Nosso SDK é compatível com .NET versão 3.5 ou maior.
...
 
 ```csharp
 MercadoPago.SDK.SetAccessToken = "ENV_ACCESS_TOKEN";
 ```
 
-3. Usando resource objects
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/sdks/official/dotnet.pt.md:51]:
```suggestion
```

File: guides/sdks/official/java.en.md [hunk 25]
```diff
@@ -32,16 +32,18 @@ Our SDK is compatible with Java 6 or grater. Our development team recomends to u
...
-    ```
-  - **For API or Custom-checkout:**
+ 
     ```java
       MercadoPago.SDK.setAccessToken("ACCESS_TOKEN");
     ```
+> NOTE
+>
+> Note
+>
```
Comment (adfonseca) [inline guides/sdks/official/java.en.md:44]:
```suggestion
```

File: guides/sdks/official/java.es.md [hunk 26]
```diff
@@ -24,24 +24,27 @@ Nuesto SDK es compatible con las versiones de Java 8 o superior.
...
+
     ```java
       MercadoPago.SDK.setAccessToken("ACCESS_TOKEN");
     ```
-3. Usando las clases del SDK.
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/sdks/official/java.es.md:44]:
```suggestion
```

File: guides/sdks/official/java.pt.md [hunk 27]
```diff
@@ -23,22 +24,25 @@ Nosso SDK é compatível com Java versão 8 ou maior.
...
+
     ```java
       MercadoPago.SDK.setAccessToken("ACCESS_TOKEN");
     ```
-3. Usando objetos de recursos.
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/sdks/official/java.pt.md:42]:
```suggestion
```

File: guides/sdks/official/js.es.md [hunk 28]
```diff
@@ -14,7 +14,13 @@ Como primer paso, tienes que incluir el SDK en el HTML de tu aplicación:
 
 ##Configura tu clave pública
 
-Para comenzar a utilizar el SDK Javascript, debes proveer tú public key para que podamos identificarte al conectarte con MercadoPago.
+Para comenzar a utilizar el SDK Javascript, debes proveer tú [Public key]([FAKER][CREDENTIALS][URL]) para que podamos identificarte al conectarte con MercadoPago.
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/sdks/official/js.es.md:22]:
```suggestion
```

File: guides/sdks/official/js.pt.md [hunk 29]
```diff
@@ -14,7 +14,13 @@ Como primeiro passo, deve-se incluir o SDK no HTML da sua aplicação:
 
 ##Configure a sua chave pública
 
-Para começar a utilizar o SDK Javascript, deve-se prover a sua public key para que possamos identificá-lo ao se conectar com o MercadoPago.
+Para começar a utilizar o SDK Javascript, deve-se prover a sua [Public key]([FAKER][CREDENTIALS][URL]) para que possamos identificá-lo ao se conectar com o MercadoPago.
+
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/sdks/official/js.pt.md:22]:
```suggestion
```

File: guides/sdks/official/php.en.md [hunk 30]
```diff
@@ -28,20 +28,17 @@ This SDK supports PHP version 5.6 or newer.
...
-  * **For API or custom checkout:**
     ```php
     MercadoPago\SDK::setAccessToken("ACCESS_TOKEN");      // On Production
     MercadoPago\SDK::setAccessToken("TEST_ACCESS_TOKEN"); // On Sandbox
     ```
 
+> NOTE
+>
+> Note
+>
```
Comment (adfonseca) [inline guides/sdks/official/php.en.md:39]:
```suggestion
```

File: guides/sdks/official/php.es.md [hunk 31]
```diff
@@ -10,39 +10,36 @@ Nuesto SDK es compatible con las versiones de PHP 5.6 o superior.
...
     ```php
     MercadoPago\SDK::setAccessToken("ACCESS_TOKEN");      // On Production
     MercadoPago\SDK::setAccessToken("TEST_ACCESS_TOKEN"); // On Sandbox
     ```
 
-3. Usando los objetos del SDK.
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/sdks/official/php.es.md:39]:
```suggestion
```

File: guides/sdks/official/php.pt.md [hunk 32]
```diff
@@ -10,39 +10,36 @@ Nosso SDK é compatível com PHP versão 5.6 ou maior.
...
     ```php
     MercadoPago\SDK::setAccessToken("ACCESS_TOKEN");      // On Production
     MercadoPago\SDK::setAccessToken("TEST_ACCESS_TOKEN"); // On Sandbox
     ```
 
-3. Usando objetos de recursos.
+> NOTE
+>
+> Nota
+>
```
Comment (adfonseca) [inline guides/sdks/official/php.pt.md:39]:
```suggestion
```

File: guides/plugins/official/magento-two.es.md [hunk 33]
```diff
@@ -57,21 +57,13 @@ Este proceso explicará cómo configurar el módulo para aceptar pagos con Check
...
-* Uruguay: [https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago.com/mlu/account/credentials]([FAKER][CREDENTIALS][URL])
-* Venezuela: [https://www.mercadopago.com/mlv/account/credentials](https://www.mercadopago.com/mlv/account/credentials)
-* Peru: [https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago.com/mpe/account/credentials]([FAKER][CREDENTIALS][URL])
+2) Para configurar las credenciales acceda a la opción **Mercado Pago** > **Credenciales**, aparecerá el campo **Public key** y el campo **Access token**. Obtén tus credenciales en la sección [Credenciales]([FAKER][CREDENTIALS][URL]).
 
 
 > Existen dos tipos de credenciales:
 > * Modo Sandbox: Las credenciales de este modo se utilizan para realizar pruebas.
 > * Modo Produção: Las credenciales de este modo se utilizan para recibir los pagos en Producción. Para utilizar las credenciales del modo de producción debe completar el formulario de "Quiero ir a producción"
```
Comment (adfonseca) [inline guides/plugins/official/magento-two.es.md:65]:
```suggestion
> * Modo Produção: Las credenciales de este modo se utilizan para recibir los pagos en Producción. Para utilizar las credenciales del modo de producción debe completar el formulario de "Quiero ir a producción".
```

File: guides/plugins/official/magento-two.es.md [hunk 34]
```diff
@@ -57,21 +57,13 @@ Este proceso explicará cómo configurar el módulo para aceptar pagos con Check
...
-* Peru: [https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago.com/mpe/account/credentials]([FAKER][CREDENTIALS][URL])
+2) Para configurar las credenciales acceda a la opción **Mercado Pago** > **Credenciales**, aparecerá el campo **Public key** y el campo **Access token**. Obtén tus credenciales en la sección [Credenciales]([FAKER][CREDENTIALS][URL]).
 
 
 > Existen dos tipos de credenciales:
 > * Modo Sandbox: Las credenciales de este modo se utilizan para realizar pruebas.
-> * Modo Produção: Las credenciales de este modo se utilizan para recibir los pagos en Producción. Para utilizar las credenciales del modo de producción debe completar el formulario de "Quiero ir a producción"
+> * Modo Produção: Las credenciales de este modo se utilizan para recibir los pagos en Producción. Para utilizar las credenciales del modo de producción debe completar el formulario de "Quiero ir a producción".
+Encuentra toda la información sobre tus credenciales en nuestras [preguntas frecuentes](https://www.mercadopago.com.ar/developers/es/guides/faqs/credentials/).
```
Comment (adfonseca) [inline guides/plugins/official/magento-two.es.md:66]:
```suggestion
> * Encuentra toda la información sobre tus credenciales en nuestras [preguntas frecuentes](https://www.mercadopago.com.ar/developers/es/guides/faqs/credentials/).
```

File: guides/plugins/official/magento-two.pt.md [hunk 35]
```diff
@@ -77,21 +77,13 @@ Esse processo irá explicar como configurar o módulo para aceitar pagamentos co
...
-* Venezuela: [https://www.mercadopago.com/mlv/account/credentials](https://www.mercadopago.com/mlv/account/credentials)
-* Peru: [https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago.com/mpe/account/credentials]([FAKER][CREDENTIALS][URL])
+2) Para configurar as credenciais acesse a opção **Mercado Pago** > **Credentials**, irá aparecer o campo da **Public key** e o campo do **Access token**. Obtenha suas credenciais na seção [Credenciais]([FAKER][CREDENTIALS][URL])
 
 
 > Existem dois tipos de credenciais:
 > * Modo Sandbox: As credenciais desse modo são utilizadas para realizar testes.
 > * Modo Produção: As credenciais desse modo são utilizadas para receber os pagamentos em Produção. Para utilizar as credenciais do Modo Produção é necessário preencher o formulário "Eu quero ir para produção".
+Encontre toda a informação sobre suas credenciais em nossas [perguntas frequentes](https://www.mercadopago.com.br/developers/pt/guides/faqs/credentials/).
```
Comment (adfonseca) [inline guides/plugins/official/magento-two.pt.md:86]:
```suggestion
> * Encontre toda a informação sobre suas credenciais em nossas [perguntas frequentes](https://www.mercadopago.com.br/developers/pt/guides/faqs/credentials/).
```

**Pull request merged**